### PR TITLE
feat: update pricing page

### PR DESCRIPTION
**Pull request by ecmadao**
图片

### CODE REVIEW

File: pages/pricing/index.vue [hunk 1]
```diff
@@ -70,36 +78,28 @@
...
-                  ]"
+                  class="text-gray-600 ml-3 text-sm font-medium"
                 >{{ feature }}</span>
               </li>
             </ul>
           </div>
         </div>
       </div>
+      <div class="max-w-7xl mx-auto px-4 py-4 text-right text-gray-400">
+        You can upgrade, downgrade, or cancel your subscription anytime. No contracts, no hidden charges.
```
Comment (tianzhou) [inline pages/pricing/index.vue:101]: Option 1:

make this text centered and leave some top space. So it looks like it sits in the middle. The reason is we have action button both above and below, this text can help both regardless of which one the user looks at.

Option 2, move this text above the illustration and center the text, change the wording slightly.

Upgrade, downgrade or cancel your subscription anytime. No hidden charges.

File: pages/pricing/index.vue [hunk 2]
```diff
@@ -315,8 +342,34 @@
...
+              ]"
+              @click="onButtonClick(plan)"
+            >{{ plan.buttonText }}</button>
+          </div>
+        </div>
+
       </div>
     </section>
+    <div class="max-w-7xl mx-auto px-4 py-4 pb-24 text-right text-gray-400">
+      You can upgrade, downgrade, or cancel your subscription anytime. No contracts, no hidden charges.
```
Comment (tianzhou) [inline pages/pricing/index.vue:371]: let's remove the "No contracts". I guess we may still have it.

File: pages/pricing/index.vue [hunk 3]
```diff
@@ -294,6 +320,7 @@
                           }}
                         </span>
                       </template>
+                      <span v-if="tier.tooltip" class="tooltip whitespace-nowrap">{{ tier.tooltip }}</span>
```
Comment (tianzhou) [inline pages/pricing/index.vue:323]: Can we add a tooltip button and only hover on the button shows the tooltip?

File: common/plan.ts [hunk 4]
```diff
@@ -132,11 +137,11 @@ const TEAM_PLAN: Plan = {
 
 const ENTERPRISE_PLAN: Plan = {
   // Plan meta data
+  version: "Beta",
```
Comment (tianzhou) [inline common/plan.ts:140]: As a 2nd thought, let's change this to "Contact us"

File: pages/pricing/index.vue [hunk 5]
```diff
@@ -50,18 +58,18 @@
...
                 <div class="mt-3 flex items-baseline">
                   <p
-                    :class="[
-                      plan.featured ? 'text-indigo-600' : 'text-white',
-                      'text-4xl font-extrabold tracking-tight',
-                    ]"
+                    class="text-4xl font-extrabold tracking-tight"
                   >{{ plan.unitPrice }}</p>
                 </div>
+                <p :class="plan.featured ? '' : 'opacity-0 disabled'">
```
Comment (tianzhou) [inline pages/pricing/index.vue:64]: This should be reversed, highlight the instance/month price and use a small and gray color for the /year price

File: common/plan.ts [hunk 6]
```diff
@@ -132,11 +137,11 @@ const TEAM_PLAN: Plan = {
...
   type: PlanType.ENTERPRISE,
   trialDays: 7,
   unitPrice: "0",
```
Comment (tianzhou) [inline common/plan.ts:143]: let's make enterprise $199 per month.

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Comment by ecmadao** [comment]
Updated
图片

**Comment by ecmadao** [comment]
Updated
图片

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**